core/src/components/input/input.scss

```diff
@@ -257,22 +229,6 @@
 }
 
 
-// Input Has focus
-// --------------------------------------------------
-
-// TODO FW-2764 Remove this
-:host(.has-focus.legacy-input) {
-  pointer-events: none;
-}
-
-// TODO FW-2764 Remove this
-:host(.has-focus.legacy-input) input,
-:host(.has-focus.legacy-input) a,
-:host(.has-focus.legacy-input) button {
-  pointer-events: auto;
-}
```

Comment:
Out of curiosity, can the other `:host-context` stuff in this file be removed too?

Reply:
Good catch! Removed here and looks like everything is still working properly: https://github.com/ionic-team/ionic-framework/pull/29019/commits/d1a7adc1aba3d544b5d829f548fd63b2ccb29b6f

This should also be removed from `textarea`: https://github.com/ionic-team/ionic-framework/blob/feature-8.0/core/src/components/textarea/textarea.ios.scss#L19-L20

We could just do this when we go through `item` though, I added a note on FW-5289